metadata.yaml

```diff
@@ -0,0 +1,27 @@
+name: jenkins-k8s-operator
+display-name: Jenkins K8s
+summary: Jenkins Continuous Integration Server
+maintainers:
+  - IS SRE <launchpad.net/~canonical-is-devops>
+description: >
```

Comment:
The description should match the format and contents described in https://discourse.charmhub.io/t/changes-to-charmhub-io-pages/9366